doc_files/iter8_bookinfo_istio.md

```diff
@@ -424,10 +424,10 @@ kubectl apply -n iter8 -f https://raw.githubusercontent.com/iter8-tools/iter8-co
 #### Note:
 > For additional information about how to add a new metric to the existing configuration please see [this documentation](metrics.md).
 
-To verify that the new metric has been added to the configmap, you can check it again:
+To verify that the new metric has been added to the ConfigMap, you can check it again:
 
 ```bash
-$ kubectl get configmap iter8config-metrics -n iter8 -oyaml
+kubectl get configmap iter8config-metrics -n iter8 -oyaml
```

Comment:
All other bash commands do not have the preceding $.

Only shell session examples use the $.